src/main/resources/default/taglib/t/directoryField.html.pasta

```diff
@@ -0,0 +1,69 @@
+<i:arg name="name" type="String" default=""/>
+<i:arg name="fieldName" type="String" default="@name"/>
+<i:arg name="value" type="String"/>
+<i:arg name="basePath" type="String" default=""
+       description="Provides a default value which is used to the select box"/>
```

Comment:
Improve the description. Is this the initial path the file picker is rendered with, or what does it do exactly?